src/components/key-stats/colours.css

```diff
@@ -0,0 +1,6 @@
+:root {
```

Comment:
would it be better to keep all colours, primary and secondary in the same place? Just to make managing them a bit easier, I think these are used outside key stats components too

Reply:
yeah, I'm happy to move this up to a higher level – like I said on #45, we've taken two different approaches to this, so maybe we're better off getting both PRs into master and then unifying

Reply:
yep sounds good 👍